rule/use_errors_new.go

```diff
@@ -0,0 +1,60 @@
+package rule
+
+import (
+	"go/ast"
+
+	"github.com/mgechev/revive/lint"
+)
+
+// UseErrorsNewRule lints given else constructs.
```

Comment:
I don't get this sentence, maybe you cluld  rephrase

Reply:
done